src/contractTest/java/uk/gov/hmcts/reform/em/npa/consumer/NpaPactRedactionConsumerTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.reform.em.npa.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.LambdaDsl;
+import au.com.dius.pact.consumer.dsl.PactBuilder;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.V4Pact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import lombok.extern.slf4j.Slf4j;
+import net.serenitybdd.rest.SerenityRest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static org.hamcrest.Matchers.notNullValue;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@Slf4j
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class NpaPactRedactionConsumerTest {
+
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+    public static final String AUTH_TOKEN = "Bearer someAuthorizationToken";
+    public static final String SERVICE_AUTH_TOKEN = "Bearer someServiceAuthorizationToken";
+
+    private static final UUID REDACTION_ID = UUID.fromString("4c34ba4a-585a-407d-aa78-3f86f3171cdd");
+    private static final UUID DOCUMENT_ID = UUID.fromString("f2cc4d79-d0f3-4b43-affe-535516370cdd");
+
+    public Map<String, String> getHeaders() {
+        return Map.of(
+                SERVICE_AUTHORIZATION, SERVICE_AUTH_TOKEN,
+                AUTHORIZATION, AUTH_TOKEN,
+                "Content-Type", "application/json"
+        );
+    }
+
+    @Pact(consumer = "em_npa_api", provider = "native_pdf_annotator_api_redaction_provider")
```

Comment:
Since this is now its own consumer class, we should rename the consumer so it's unique and specific, "em_npa_redaction_api" for example. Also prevents failure scenarios when different consumers use the same name